tests/com/tony/server/AuthenticatorTest.java

```diff
@@ -0,0 +1,83 @@
+package com.tony.server;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.*;
```

Comment:
Don't use wildcard imports, it's better to be explicit with what you are bringing in.